src/verilog/verilog_typecheck_expr.cpp

```diff
@@ -1023,19 +1025,23 @@ exprt verilog_typecheck_exprt::convert_symbol(symbol_exprt expr)
       return std::move(expr);
     }
   }
-  else if(!implicit_wire(identifier, symbol))
-  {
-    // this should become an error
-    warning().source_location=expr.source_location();
-    warning() << "implicit wire " << symbol->display_name() << eom;
-    expr.type()=symbol->type;
-    expr.set_identifier(symbol->name);
-    return std::move(expr);
-  }
   else
   {
-    throw errort().with_location(expr.source_location())
-      << "unknown identifier " << identifier;
+    if(implicit_net_type.has_value())
+    {
+      implicit_wire(identifier, symbol, implicit_net_type.value());
+
+      warning().source_location = expr.source_location();
+      warning() << "implicit wire " << symbol->display_name() << eom;
```

Comment:
Why does this yield a warning, or, rather: in what way should the input have been modified?

Reply:
If this is covered by the standard (as the commit message suggests), why is such a recommendation warranted?

Reply:
ok, will drop